alphalens/performance.py

```diff
@@ -966,6 +966,9 @@ def average_cumulative_return(q_fact, demean_by):
             #
             avgcumret = g_fq.groupby(g_fq).apply(average_cumulative_return,
                                                  demean_by)
+            if len(avgcumret) == 0:
```

Comment:
Is this the fix for the weird crash?

Reply:
No, that's another small bug fix for a very corner case scenario that I discovered while debugging the original bug :) But each fix is in a separate commit, so the git history is clean. Sorry for including everything in this PR but the fixes are so small that I didn't bother to create separate PRs for each small issue.